spec/factories/quota_factory.rb

```diff
@@ -20,6 +25,35 @@
     trait :xml do
       validity_end_date { Date.current.ago(1.year) }
     end
+
+    trait :current do
+      validity_end_date { nil }
+    end
+
+    trait :non_current do
```

Comment:
`:historical` or `:expire` might make more sense here?

Reply:
I prefer expire over historical. Happy to change it :)